alias

```diff
@@ -134,3 +134,11 @@ revoke-my-ip-all =
   !f() {
     aws revoke-my-ip ${1} all all
   }; f
+
+get-elbv2-dnsname =
+  !f() {
+    dnsname=$(aws elbv2 describe-load-balancers --names ${1} --query 'LoadBalancers[].DNSName' --output text)
+    zone_id=$(aws route53 list-hosted-zones-by-name --dns-name ${2} --query "HostedZones[?Name=='${2}.'].Id" --output text | cut -d/ -f3)
+    echo "Looking up ${1} in Hosted Zone ${2} ($zone_id):"
+    aws route53 list-resource-record-sets --hosted-zone-id ${zone_id} --query "ResourceRecordSets[?AliasTarget.DNSName=='${dnsname}.'].Name" --output json | jq -r '.[]'
```

Comment:
- [x] Using `${dnsname,,}` to force lowercase and get a match.
- [x] Using curly braces to output 1 resource record per line

```suggestion
	aws route53 list-resource-record-sets --hosted-zone-id "${zone_id}" --query "ResourceRecordSets[?AliasTarget.DNSName=='${dnsname,,}.'].{LBHosts:Name}" --output text
```